src/components/macro/edit/macro-edit.component.scss

```diff
@@ -1,5 +1,11 @@
-.not-found {
-    margin-top: 30px;
-    font-size: 16px;
-    text-align: center;
+:host {
+    width: 100%;
+    height: 100%;
+    display: block;
+
+    .not-found {
```

Comment:
Because it will reduce time in style step in pixel pipeline. That's why BEM is good, because you have one level only, so page is rendering faster. Nesting is no good 😄